webcheck.go

```diff
@@ -110,39 +108,62 @@ func checkBodyReaderMatchesPattern(reader io.Reader, pattern string, expectedPre
 	return nil
 }
 
-func (fm *Frontman) runWebCheck(data WebCheckData) (map[string]interface{}, error) {
-	prefix := fmt.Sprintf("http.%s.", data.Method)
+func (check WebCheck) uniqueID() string {
+	return check.UUID
+}
+
+func (check WebCheck) run(fm *Frontman) (*Result, error) {
+
+	res := &Result{
```

Comment:
code repetition see a similar issue above

Reply:
It's due to them being different types.